### PR TITLE
Add StartPaused & CombatUnpauses Features

### PR DESCRIPTION
**Pull request by prikhi**
StartPaused pauses the timer when initializing & reseting the meter.
CombatUnpauses automatically unpauses the timer when entering combat.

### CODE REVIEW

File: MXPV.lua [hunk 1]
```diff
@@ -452,6 +470,9 @@ function MXPV.GetSetCombatState (event, inCombat)
 			MXPV.Meter.Pause()
 		end
 	end
+	if MXPV.SavedVars.CombatUnpauses and MXPV.Meter.Paused then
```
Comment (lapinott) [inline MXPV.lua]: Should be <code>if MXPV.SavedVars.CombatUnpauses and MXPV.Meter.Paused and inCombat</code> or I guess <code>MXPV.Meter.Start()</code> will fire when exiting combat too.

Comment (prikhi) [inline MXPV.lua]: Fixed.

Comment (lapinott) [inline MXPV.lua]: Actually full code should be:

<code>elseif MXPV.Meter.Paused and MXPV.SavedVars.CombatUnpauses and inCombat MXPV.Meter.Start()
end
</code>

File: MXPV.lua [hunk 2]
```diff
@@ -452,6 +470,9 @@ function MXPV.GetSetCombatState (event, inCombat)
 			MXPV.Meter.Pause()
 		end
 	end
+	if MXPV.SavedVars.CombatUnpauses and MXPV.Meter.Paused and inCombat then
```
Comment (lapinott) [inline MXPV.lua]: <code>elseif</code> :)

File: MXPV.lua (hunk 1 above)
Comment (prikhi) [inline MXPV.lua]: OK, wasn't sure if I should merge the if blocks or keep them discrete. Fixed again.

Comment (lapinott) [inline MXPV.lua]: Yes well I think this option only needs to be handled when not in 
CombatOnly mode -> meter will start anyways then.

**Pull request merged**